app/src/logger/DiscordTransport.ts

```diff
@@ -0,0 +1,76 @@
+import Transport from 'winston-transport';
+import axios, {AxiosInstance, AxiosProxyConfig} from 'axios';
+import {get} from 'lodash';
+
+export default class DiscordTransport extends Transport {
+    private readonly webhookUrl: string;
+    private readonly username?: string;
+    private readonly avatarUrl?: string;
+
+    private readonly axios: AxiosInstance;
+
+    constructor(opts: DiscordTransportOptions) {
+        super(opts);
+
+        this.webhookUrl = opts.webhookUrl;
+        this.username = opts.username;
+        this.avatarUrl = opts.avatarUrl;
```

Comment:
`avatarUrl` and `username` don't appear to be injected anywhere, wouldn't that error?

Reply:
No, they are optional as you can set them when creating a webhook in discord